.buildkite/pipeline.yml

```diff
@@ -583,3 +583,43 @@ steps:
     build:
       commit: "${BUILDKITE_COMMIT}"
       branch: "${BUILDKITE_BRANCH}"
+
+  - label: "Trigger Agentbeat"
+    if: build.pull_request.id != null
+    plugins:
+      - monorepo-diff#v1.0.1:
+          diff: "git diff --name-only origin/${GITHUB_PR_TARGET_BRANCH}...HEAD"
+          watch:
+            - path:
+                - auditbeat/
+                - filebeat/
+                - heartbeat/
+                - metricbeat/
+                - osquerybeat/
+                - packetbeat/
+
+                - x-pack/auditbeat/
+                - x-pack/filebeat/
+                - x-pack/heartbeat/
+                - x-pack/metricbeat/
+                - x-pack/osquerybeat/
+                - x-pack/packetbeat/
```

Comment:
do you need to add each directory or could you just add `- x-pack/`? If not why is `x-pack/agentbeat` not here?

Reply:
Do we need to run it if any x-pack subdirectory is changed?

Reply:
I guess that is true, we don't need all of them.